pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: memoize
-version: 2.0.0
+version: 3.0.0
```

Comment:
This update is forward compatible, maybe we shouldn't update the major version number?

Reply:
No, let's keep this a major change.